packages/documentation/src/content/docs/playground/overview.md

```diff
@@ -135,36 +135,37 @@ pnpm localenv:compose down --volumes
 
 ### Usage
 
-#### Postman & Open Payments APIs
+#### Bruno & Open Payments APIs
 
-The Open Payments APIs can be interacted with using the [Postman API collection](https://www.postman.com/interledger/workspace/interledger/api/84fc90ca-3153-4865-8b49-b91218e5d574). You can either log into Postman and fork the collection or you can [import it from source](https://raw.githubusercontent.com/interledger/rafiki/main/postman/collections/Interledger.json). It requires you to either load the [Local Environment](https://raw.githubusercontent.com/interledger/rafiki/main/postman/environments/Local%20Playground%20Environment.postman_environment.json) or the [Remote Environment](https://raw.githubusercontent.com/interledger/rafiki/main/postman/environments/Remote%20Environment.postman_environment.json).
+The Open Payments APIs can be interacted with using the [Bruno collection](https://github.com/interledger/rafiki/main/bruno/collections/Interledger) ([resource server endpoints](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20APIs) and [auth server endpoints](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20Auth%20APIs)). It requires you to
 
-The Examples folder in the Postman API collection includes an eCommerce (Open Payments) example that can be executed one by one. It
+1. load the collection into Bruno by clicking "Open Collection"
+2. navigating to `/rafiki/main/bruno/collections/Interledger` on your machine and
+3. clicking "Open"
+   Furthermore, you need to either load the [Local Environment](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/environments/Local%20Playground.bru) or the [Remote Environment](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/environments/Remote.bru).
 
-1. gets the sender's wallet address
-2. gets the receiver's wallet address
+The Examples folder in the Bruno collection includes an [Open Payments](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Examples/Open%20Payments) example that can be executed one by one. It
+
+1. requests the sender's wallet address
+2. requests the receiver's wallet address
 3. requests a grant to create an incoming payment on the receiver's account
-4. creates an incoming payment on the receiver's account
-5. requests a grant to create (and read) a quote on the senders's account
+4. creates an incoming payment on receiver's account
+5. requests a grant to create (and read) a quote on the sender's account
 6. creates a quote on the sender's account
 7. requests a grant to create (and read) an outgoing payment on the sender's account
-8. creates an outgoing payment on the sender's account
-9. fetches the outgoing payment on the sender's account
+8. continues the grant request (via the interaction flow)
```

Comment:
should we just say about copying over the `interaction_ref`? 

What about changing the outgoing payment grant request altogether to _not_ have a finish method, and just be able to call continue request without needing to enter the `interaction_ref` using the new polling method?